### ISSUE TITLE
Is it possible to statically link Allegro5 into a project?

### ISSUE THREAD

**Issue by StathisKap** [liballeg/allegro5#1347]

**Comment by EdgarReynaldo** [comment]
Yes, you can static link allegro 5. Assuming you're on Windows (you didn't specify) you link to the static allegro monolith and all of its dependencies or if using VS link to the static version of allegro with Nuget.

**Comment by StathisKap** [comment]
Hey, yeah sorry for not specifying. I'm on Macos (M1). 
How would I got about doing it then?

**Comment by EdgarReynaldo** [comment]
This link might help you. You need to link the static fframeworks. https://curtis.schlak.com/2014/05/22/creating-and-using-allegro-5-static-libraries-with-xcode-5.html
